src/utils/dropdown/optionList.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import {
+  Dimensions,
+  StyleSheet,
+  View,
+} from 'react-native';
+import Overlay from './overlay';
+import Items from './items';
+
+const window = Dimensions.get('window');
+
+class OptionList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      show: false,
+
+      width: 0,
+      height: 0,
+
+      pageX: 0,
+      pageY: 0,
+
+      positionX: 0,
+      positionY: 0,
+
+      items: [],
+      seleted: null,
+      onSelect: () => { },
+    };
+  }
+
+  currentPosition(pageX, pageY) {
+    this.setState({
+      ...this.state,
+      pageX,
+      pageY,
+    });
+  }
+
+  show(items, selected, positionX, positionY, width, height, onSelect) {
+    let h = (items.length > 3) ? height * 3 + height / 4 : height * (items.length + 1);
```

Comment:
3 means showing line, and 4 is remain of next option for easy to recognize scrollable